bin/ci/script.sh

```diff
@@ -30,6 +30,15 @@ if [[ "$ORCA_ENABLE_NIGHTWATCH" == "TRUE" && "$ORCA_SUT_HAS_NIGHTWATCH_TESTS" &&
       --passWithNoTests \\
       --tag=$ORCA_SUT_MACHINE_NAME"
 
+    if [[ "$ORCA_SUT_NAME" == "drupal/example" ]]; then
```

Comment:
This is internal/self-test code. We don't want it in the customer-facing `script.sh`. It should be in `bin/ci/self-test/script.sh`.